### DISCUSSION TITLE
Could someone share the code used to build the StandardReportCountry html file?

### DISCUSSION THREAD

**Discussion by Mahavirinvestments** [Gilead-Public/gsm#1507] [Q&A] [answered]
I like the look of it and am having trouble replicating it on my own.

**Comment by mattroumaya** [comment, marked as answer]
Hi @Mahavirinvestments - sorry for a delayed response!

- You can find the code that generates the `.Rmd` file [here](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/report/KRIReportBySite.Rmd).
- The `.Rmd` file above uses a lot of the functions found in [`util-ReportHelpers.R`](https://github.com/Gilead-BioStats/gsm/blob/dev/R/util-ReportHelpers.R).
- If you like the look of the styling of the report itself,  you can also check out [`styles.css`](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/report/styles.css).

Hope this is helpful - please add some more comments if you'd like to discuss anything.
